kivy/utils.py

```diff
@@ -113,14 +113,36 @@ def rgba(s, *args):
 def get_color_from_hex(s):
     '''Transform a hex string color to a kivy
     :class:`~kivy.graphics.Color`.
+
+    .. versionadded:: 2.1.0
+
+    Added support and for 3-4 HEX values, because :mod:`pygments.styles`
+    also support this and also because is common on web development.
+
+    .. note::
+        If provided values has 1-2 characters, that will be treated
+        as a grey scale.
+
+    >>> get_color_from_hex("#009933FF")     # [0.0, 0.6, 0.2, 1.0]
+    >>> get_color_from_hex("093")           # [0.0, 0.6, 0.2, 1.0]
+    >>> get_color_from_hex("#FF")           # [1.0, 1.0, 1.0, 1.0]
+    >>> get_color_from_hex("0")             # [0.0, 0.0, 0.0, 0.0]
     '''
-    if s.startswith('#'):
-        return get_color_from_hex(s[1:])
+    # Remove the usually HEX prefix
+    if s.startswith('#'): return get_color_from_hex(s[1:])
```

Comment:
Why a one-liner?

Reply:
Is just a matter of code style. I just copy-pasted my edited kivy source code. I made some few changes to feet my project needs.